.travis.yml

```diff
@@ -4,12 +4,16 @@ node_js:
   - "12"
 
 before_install:
-  - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.19.1
+  - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.21.1
   - export PATH=$HOME/.yarn/bin:$PATH
 
 before_script:
   - yarn run build
 
 cache:
   directories:
-    - $HOME/.yarn
\ No newline at end of file
+    - $HOME/.yarn
+
+script:
+  - node_modules/.bin/eslint src/**/*.ts
```

Comment:
Do we want to run it for test files too?

Reply:
Yup sounds good